src/components/global/Playground/index.tsx

```diff
@@ -182,19 +185,77 @@ export default function Playground({
     return `playground_${id}_${fileNameId}`;
   }
 
+  function getFileIcon(fileName: string) {
```

Comment:
Can we move these SVGs to separate files that we import? The code of this component is getting a bit hairy with all the SVG data inlined.